### DISCUSSION TITLE
Shipping a package that only contains CMake utilities?

### DISCUSSION THREAD

**Discussion by Holt59** [microsoft/vcpkg#39768] [Q&A]
I want to create a port for a "package" that only exposes CMake functions. My current "workaround" can be found here: https://github.com/ModOrganizer2/vcpkg-registry/tree/main/ports/mo2-cmake, with `mo2-cmake-config.cmake` including the other `.cmake` file and exposing function.

I can then add the dependency as a non-host dependency and bring functions with

```cmake
find_package(mo2-cmake CONFIG REQUIRED)
```

Is there a better way to package such utilities?

**Comment by Osyotr** [comment]
This is how it's done for CMakeRC: https://github.com/microsoft/vcpkg/blob/a1d682b5aa5385c849a2beb0e1203757b9625d41/ports/cmakerc/portfile.cmake
